src/services/apes/apes.ts

```diff
@@ -46,31 +47,27 @@ export const apes = createApi({
   endpoints: (builder) => ({
     capturePayPalOrder: builder.mutation<PayPalOrder, { orderId: string }>({
       query: (params) => ({
-        url: `v1/fiat/paypal/apes/${apiEnv}/orders/${params.orderId}/capture`,
+        url: `v1/fiat/paypal/${apiEnv}/orders/${params.orderId}/capture`,
         method: "POST",
         headers: { authorization: TEMP_JWT },
       }),
     }),
-    createPayPalOrder: builder.mutation<
-      { orderId: string },
-      CreatePayPalOrderParams
-    >({
+    paypalOrder: builder.query<string, CreatePayPalOrderParams>({
       query: (params) => ({
-        url: `v1/fiat/paypal/apes/${apiEnv}/orders`,
+        url: `v1/fiat/paypal/${apiEnv}/orders`,
         method: "POST",
         headers: { authorization: TEMP_JWT },
         body: JSON.stringify(params),
       }),
+      transformResponse: (res: { orderId: string }) => res.orderId,
     }),
-    createStripePaymentIntent: builder.mutation<
-      { clientSecret: string },
-      StripePaymentIntentParams
-    >({
+    stripePaymentIntent: builder.query<string, StripePaymentIntentParams>({
```

Comment:
`POST` inside of "query" looks weird at first but this setup works and has these advantages:
1. payment intent can be created when page loads
2. as this is a query, intent can be cached for up to [60s](https://redux-toolkit.js.org/rtk-query/usage/cache-behavior#default-cache-behavior:~:text=(default%2060%20seconds)). And would only revalidate if user changed `StripePaymentIntentParams` - amount, currency...etc.